Document our tracing feature (#2832)
giantswarm/docs: src/content/getting-started/observe-your-clusters-and-apps/index.md

- **Platform metrics**: CPU, memory, disk, and network metrics from all cluster nodes
- **Kubernetes metrics**: Pod status, deployments, services, and cluster events
- **Application logs**: Automatic collection from all pods in your cluster

> - **Distributed tracing**: Optional trace collection for understanding request flows (requires enablement)

Author: ```suggestion
- **Distributed tracing**: Trace collection for understanding request flows. To enable on your clusters, ask your account engineer.
```

**Want to learn more?** Our [Advanced LogQL Tutorial]({{< relref "/overview/observability/data-management/data-exploration/advanced-logql-tutorial" >}}) covers complex log queries and analysis.

## Step 5: Set up distributed tracing (Optional)

Distributed tracing helps you understand how requests flow through your microservices architecture. Giant Swarm's observability platform supports tracing through Tempo and OpenTelemetry.

> **Important**: Tracing is available in alpha from cluster release v31 and fully supported from v33. In order to use it in your installation, it must be enabled. You can request tracing in your observability platform by talking to your account engineer. 

Author: ```suggestion
**Important**: Tracing is available in alpha from cluster release v31 and fully supported from v33. In order to use it in your installation, it must be enabled. You can request tracing in your observability platform by talking to your account engineer.
```

### Prerequisites for tracing

- **Cluster version**: v31+ (alpha) or v33+ (full support)
- **Tracing enabled**: Contact your Account Engineer to enable tracing for your cluster
- **Application instrumentation**: Your applications must be instrumented with OpenTelemetry

### Configure trace collection

Once tracing is enabled, configure your applications to send traces to the cluster-local OTLP endpoint:

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: my-traced-app
spec:
  template:
    metadata:
      labels:
        # Required for trace data routing
        observability.giantswarm.io/tenant: my-team
        app: my-traced-app
    spec:
      containers:
      - name: app
        image: my-app:latest
        env:
        - name: OTEL_EXPORTER_OTLP_ENDPOINT
          value: "http://otlp-gateway.kube-system.svc:4318"
        - name: OTEL_RESOURCE_ATTRIBUTES
          value: "service.name=my-traced-app"
```

### Explore traces with TraceQL

Once traces are flowing, explore them in Grafana:

1. **Navigate to Explore**: Select the Tempo data source
2. **Use TraceQL queries**: Search and analyze your traces

> Try these sample queries:

Author: ```suggestion
Try these example queries:
```
